src/coreclr/inc/corinfo.h

```diff
@@ -592,6 +592,8 @@ enum CorInfoHelpFunc
     CORINFO_HELP_PATCHPOINT,                // Notify runtime that code has reached a patchpoint
     CORINFO_HELP_CLASSPROFILE,              // Update class profile for a call site
 
+    CORINFO_HELP_JIT_REVERSE_PINVOKE_ENTER_TRACK_TRANSITIONS, // Transition to cooperative mode and track transitions in reverse P/Invoke prolog.
```

Comment:
Nit: This enum does not need to be stable. It is fine to add this where it logically belongs (ie next to existing `CORINFO_HELP_JIT_REVERSE_PINVOKE_ENTER`).